runelite-api/src/main/java/net/runelite/api/ScriptID.java

```diff
@@ -408,4 +408,10 @@
 
 	@ScriptArguments(integer = 6)
 	public static final int INVENTORY_DRAWITEM = 6011;
+
+	/**
+	 * Draws items in trade window
```

Comment:
This actually sets up the trade interface, draw is separate in trade_main_drawitems. I would just say it inits the trade interface